src/aq/aq_obsvec_mod.F90

```diff
@@ -210,6 +211,28 @@ subroutine aq_obsvec_mask_with_missing(self,mask)
 
 end subroutine aq_obsvec_mask_with_missing
 ! ------------------------------------------------------------------------------
+!> Mask observations and set qcflag to one where observation vectors differ by more than lambda
+subroutine aq_obsvec_threshold_check(self,other,qcflag,config)
+implicit none
+
+! Passed variables
+type(aq_obsvec),intent(inout) :: self          !< Simulated obs vector
+type(aq_obsvec),intent(inout) :: other         !< Obs vector
+type(aq_obsvec),intent(inout) :: qcflag        !< Mask
+type(fckit_Configuration),intent(in) :: config !< Filter config
+
+real(aq_real) :: threshold
+
+call config%get_or_die("threshold",threshold)
+if ((self%nobs/=other%nobs).or.(self%nlev/=other%nlev)) call abor1_ftn('aq_obsvec_mask: inconsistent sizes')
+write(*,*) 'threshold', threshold
```

Comment:
Was it a working print or should it be kept and routed to %info() ?

Reply:
Removed